### PR TITLE
Add Explore page and file structure for rendering .md files as cards

### PR DESCRIPTION
**Pull request by willyogo**
This (hopefully) enables files in the markdown folder to be rendered as cards on the Explore page. Additionally, it adds the corresponding URL routes to the next.js app at build time so users can navigate from the Explore page to a space without reoloading the page.

Note: I haven't been able to run the app locally/get past the dependency errors. I also did not mean to update the yarn.lock file and am assuming that it's botched. 

No styling has been applied yet either. This is just a proof of concept to validate the approach.

### CODE REVIEW

**Comment by hiporox** [comment]
The `yarn.lock` updates whenever you install packages, so don't worry about that

**Comment by hiporox** [comment]
I will play around with this later, but I think that this should work. I am going to change the source from the `markdown` folder to be an external source (or just part of our DB), so that we don't have to make changes to this repo and release in order to update the explore page

**Comment by hiporox** [comment]
Is there are reason you took this approach instead of the headless CMS we were talking about today?

File: src/pages/explore/index.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import Link from 'next/link';
-import { getAllMarkdownFiles } from '../../../lib/markdown';
+import { getAllMarkdownFiles } from 'src/common/lib/markdown';
```
Comment (hiporox) [inline src/pages/explore/index.js]: This should be `@/common/`

**Comment by willyogo** [comment]
> Is there are reason you took this approach instead of the headless CMS we were talking about today?

Thank you @hiporox! We chatted with a former shapeshifter that also worked at Pipe with Cam and has a lot of experience with Storyblok. After hearing what we were trying to accomplish and that we don't really need the full CMS integrated (all we really need are the cards that route users to spaces without reloading), he recommended this lighterweight approach. He did say that if we had more than 1k files in the markdown it could start to slow the app down, but since we're not planning to go past 100 anytime soon, this should work.

Great point that it's better to store these somewhere else so we don't need to deploy to prod every time we update this! Thanks for picking this up from here 🙏  lmk if we can help with anything else on this

Once you get it functioning, can take care of making it pretty.

**Comment by hiporox** [comment]
Closing in place of #80

**Pull request closed without merging**